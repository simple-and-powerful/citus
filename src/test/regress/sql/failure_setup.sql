-- disable the maintenance daemon, prevent it from making any connections
ALTER SYSTEM SET citus.distributed_deadlock_detection_factor TO -1;
ALTER SYSTEM SET citus.recover_2pc_interval TO -1;
ALTER SYSTEM set citus.enable_statistics_collection TO false;
SELECT pg_reload_conf();

SELECT citus.mitmproxy('flow.allow()');

-- add the workers
SELECT master_add_node('localhost', :worker_1_port);  -- the second worker
SELECT master_add_node('localhost', :worker_2_port + 2);  -- the first worker, behind a mitmproxy
