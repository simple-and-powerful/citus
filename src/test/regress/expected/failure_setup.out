-- disable the maintenance daemon, prevent it from making any connections
ALTER SYSTEM SET citus.distributed_deadlock_detection_factor TO -1;
ALTER SYSTEM SET citus.recover_2pc_interval TO -1;
ALTER SYSTEM set citus.enable_statistics_collection TO false;
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

SELECT citus.mitmproxy('flow.allow()');
 mitmproxy 
-----------
 
(1 row)

-- add the workers
SELECT master_add_node('localhost', :worker_1_port);  -- the second worker
                  master_add_node                  
---------------------------------------------------
 (1,1,localhost,57637,default,f,t,primary,default)
(1 row)

SELECT master_add_node('localhost', :worker_2_port + 2);  -- the first worker, behind a mitmproxy
                  master_add_node                  
---------------------------------------------------
 (2,2,localhost,57640,default,f,t,primary,default)
(1 row)

